modules/ROOT/edits/enable-inbound-traffic-self.adoc

```diff
@@ -0,0 +1,22 @@
+= Configure Domains and Inbound Traffic (On Self-Managed Kuberenetes)
+
+To enable external applications to connect to Runtime Fabric on Self-Managed Kubernetes, you must provide an ingress controller that is compatible with your Kubernetes environment. Runtime Fabric uses the ingress controller to route requests to deployed applications.
+
+Inbound traffic is enabled by default. This allows Mule applications and API gateways to receive inbound connections. You can disable inbound traffic per application in Runtime Manager when deploying or configuring an application.
+
+Runtime Fabric allows you to configure domains that serve as endpoints for your deployed applications.
```

Comment:
GLOBALLY replace "allows" with "enables" in the context of "you," per SR.
```suggestion
Runtime Fabric enables you to configure domains that serve as endpoints for your deployed applications.
```

Reply:
ORGANIZATIONALLY, we currently have...

=Configure Domains and Inbound Traffic on Self-Managed Kubernetes
== Configure Domains for Runtime Fabric on Self-Managed Kubernetes

This is not standard, because the Level-1 title names two configuration tasks, but we have a subtask for only one of them.